src/main/java/com/monitorjbl/xlsx/impl/StreamingCell.java

```diff
@@ -146,6 +246,13 @@ public Date getDateCellValue() {
   /* Not supported */
 
   /**
+   * Not supported.  Added for compatability with POI 3.12.
+   */
+  public void removeHyperlink() {
```

Comment:
That works due to a bit of an abuse of JVM's polymorphic invocation ability; the implementing class's actual parent is not considered when doing an `invokeinterface` operation, it only looks for a matching method signature on the target class. It's really weird to see it done this way because there's no guarantee that there are not other methods on the newer interface that are not included here.

The right thing to do in this case is to use the actual interface from the newer version and make sure that all implemented methods are annotated with `@Override` so there is no ambiguity. If you want the library to support POI 3.12+, you can update it in the pom.xml file. Any new methods required by the interface can just throw `NotSupportedException`s.